sources/packages/web/src/views/aest/StudentDetails.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Student Search</a
+    >
+  </h5>
+  <PDStatus :pdStatus="studentDetail.pdStatus" />
+  <full-page-container>
+    <h2 class="color-blue padding-bottom-30">Student Details</h2>
```

Comment:
I believe that we can start using the same classes that were initially created to form.io for header and all.
From UI/UX perspective, form.io and Vue are the same, so sharing the same styles would be the way to go 😉 
@ann-aot @guru-aot @dheepak-aot, does it make sense?